motor/inventory/manager.go

```diff
@@ -159,6 +158,9 @@ func (im *inventoryManager) loadInventory(inventory *v1.Inventory) error {
 		im.vault = v
 	}
 
+	// determine the vault to use for accessing credentials
+	im.resetVault()
```

Comment:
This actually fixes a bug. Non-inmemory secrets have not worked if a vault was configured in the inventory